docs/en/observability/infrastructure-threshold-alert.asciidoc

```diff
@@ -23,9 +23,8 @@ like to be notified about and then quickly create an alert in just a few clicks.
 == Inventory conditions
 
 Conditions for each alert can be applied to specific metrics relating to the inventory type you select.
-You can choose the aggregation type, the metric, and by including a warning threshold value, you can be
-alerted on multiple threshold values based on severity scores. When creating the alert, you can still get
-notified if no data is returned for the specific metric or if the alert fails to query {es}.
+You can choose the aggregation type, the metric, and by including a warning threshold value. When creating
+the alert, you can still get notified if no data is returned for the specific metric or if the alert fails to query {es}.
```

Comment:
Why does this part need to change?  I'm not sure the first sentence makes sense any longer.

Reply:
Fixed